content/W2015/W15-event-arduino-workshop.md

```diff
@@ -0,0 +1,36 @@
+Title: Arduino Day Hardware Workshop
+Date: 2015-03-15 20:30
+Category: Events
+Tags: workshop, hardware
+Slug: arduino-workshop
+Author: Elana Hashman
+Summary: Come learn about digital hardware and Arduino programming in this women-only workshop for beginners!
+
+We're pleased to announce Luxsumi Jeevananthan's hardware workshop this
+upcoming [Arduino Day](https://day.arduino.cc/)! Lux and a team of mentors will
+introduce you to building prototypes with digital hardware and the software
+needed to interact with it.  She writes,
+
+"So you want to be a kickstarter rockstar? Everyone has to start somewhere; for
+hardware, that means starting off with an Arduino version of their project.
+
+Do you know how to program, but don't even know where to start with the
+hardware? Well, never fear, because WiCS is here! On March 28th, we will be
+running a workshop on how to put together hardware components and program them
+with an Arduino!  The cost of the workshop is just $25, the price of an
+Arduino, and you'll get to take whatever you've built home with you. We'll be
+teaching you how to program LEDs and other sensors with the board using an IDE
+so that you can get started on your very own project!"
+
+You can register for the workshop and pay the $25 fee in the MathSoc Office, MC
```

Comment:
You should probably mention that registration is limited, and maybe give a soft deadline for when registration should be submitted by.